src/_scss/pages/awardV2/idv/_awardFundingSummary.scss

```diff
@@ -0,0 +1,9 @@
+.award-funding-summary {
```

Comment:
Took the css from the `award-amounts__data-content`, you could extend it. I really like that feature, or any feature to keep things `dry` but, since the header is in the `.award-funding-summary` the `nth-of-type` needs to be `2`, but there are ways around that and could extend the `award-amounts__data-content` class.
```
.award-funding-summary {
  $border-light-gray: #d5d6d8;
    .award-funding-summary__data {
        border: 1px $border-light-gray solid;
        padding: rem(8);
        display: flex;
        justify-content: space-between;
        font-size: rem(16);
        color: #525252;
        &:nth-of-type(2) {
            border-radius: rem(5) rem(5) 0 0;
            border-bottom: 0;
        }
        &:last-of-type {
            border-radius: 0 0 rem(5) rem(5);
            border-top: 0;
        }
    }
}
```